package.json

```diff
@@ -77,6 +85,7 @@
     "@oclif/plugin-help": "5.1.7",
     "@oclif/plugin-not-found": "2.2.0",
     "@oclif/plugin-plugins": "2.0.2",
+    "@salesforce/plugin-info": "^1.1.4",
```

Comment:
Can you alias this to match the other plugins? e.g. `@sf/info`

Reply:
Updated 👍 Just curious, what benefits does this give aside from being less characters?

Reply:
Allows one to use more than one version of the same package.